src/db/transactions/db.ts

```diff
@@ -111,13 +110,9 @@ export class TransactionDB {
     }
 
     const result: AnyTransaction[] = [];
-    for (
-      let i = 0;
-      i < queueIds.length;
-      i += env.__EXPERIMENTAL_REDIS_BATCH_SIZE
-    ) {
+    for (let i = 0; i < queueIds.length; i += MAX_REDIS_BATCH_SIZE) {
```

Comment:
Replacing the `__EXPERIMENTAL_REDIS_BATCH_SIZE` env var with a hard-coded value of 50k